src/common/Util.res

```diff
@@ -84,3 +84,76 @@ module Date = {
     dateTimeFormat("en-US", {"month": "short", "day": "numeric", "year": "numeric"})->format(date)
   }
 }
+
+/**
+Takes a `version` string starting with a "v" and ending in major.minor.patch or
+major.minor.patch-prerelease.identifier (e.g. "v10.1.0" or "v10.1.0-alpha.2")
+*/
+module Semver = {
```

Comment:
Any particular reason why this module doesn't reside in the `CompilerManagerHook` module? Seems like the only module where this is used, and it seems to be quite tailored to the compiler management use-case.

Also, the module description seems to rather describe `Semver.parse`.

Reply:
Moved to CompilerManagerHook